wp-job-manager-functions.php

```diff
@@ -399,7 +400,12 @@ function wp_job_manager_create_account( $args, $deprecated = '' ) {
     }
 
     // Notify
-    wp_new_user_notification( $user_id, $password );
+    if ( version_compare( $wp_version, '4.3.0', '<=' ) ) {
+    	wp_new_user_notification( $user_id, $password );
+    }
+    elseif {
```

Comment:
Noting that you're right on this and that the version check should have been `if ( version_compare( $wp_version, '4.3.1', '<' ) ) {`